be/src/vec/exec/scan/scanner_context.cpp

```diff
@@ -55,7 +55,6 @@ ScannerContext::ScannerContext(doris::RuntimeState* state_, doris::vectorized::V
           _process_status(Status::OK()),
           _batch_size(state_->batch_size()),
           limit(limit_),
-          _max_bytes_in_queue(max_bytes_in_blocks_queue_ * num_parallel_instances),
```

Comment:
your modification is too complicated, 
just rewrite it to  std::max(max_bytes_in_blocks_queue_, 1024)*num_parallel_instances

and please add comment here to explain why we need do like this.